### PR TITLE
refact(deps): bump k8s, client-go and csi-lib deps to v0.20.2 

### PR DESCRIPTION
**Pull request by iyashu**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -7,70 +7,48 @@ require (
...
+	k8s.io/api v0.20.2
 	k8s.io/apimachinery v0.20.4
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
-	k8s.io/code-generator v0.17.2
-	k8s.io/gengo v0.0.0-20190822140433-26a664648505 // indirect
+	k8s.io/code-generator v0.20.2
 	k8s.io/klog v1.0.0
-	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a // indirect
-	k8s.io/kubernetes v1.15.12
+	//k8s.io/kubernetes v1.15.12
```
Comment (akhilerm) [inline go.mod]: This can be removed.

**Review by akhilerm — COMMENTED** [review]
PR looks good. A minor comment on removing the commented out k8s dependency from go.mod as it may cause confusion.

File: go.mod (hunk 1 above)
Comment (iyashu) [reply go.mod]: done.

**Review by akhilerm — APPROVED** [review]
LGTM

**Pull request merged**